db/schema/userProfiles.ts

```diff
@@ -0,0 +1,30 @@
+import { relations } from "drizzle-orm";
+import { jsonb, pgTable, text, timestamp, uuid } from "drizzle-orm/pg-core";
+import { authUsers } from "../supabaseSchema/auth";
+
+export type AccessRole = "afk" | "core" | "nonCore";
+
+export const userProfiles = pgTable("user_profiles", {
```

Comment:
```suggestion
// Created automatically when a user is inserted via a Postgres trigger. See db/drizzle/0101_little_arclight.sql
export const userProfiles = pgTable("user_profiles", {
```

(or something - I think it's worth commenting since it won't be obvious how this works without knowing about the trigger)